build/build.sh

```diff
@@ -0,0 +1,23 @@
+export http_proxy_host=$(echo $http_proxy | awk -F':' {'print $2'} | tr -d '/')
+export http_proxy_port=$(echo $http_proxy | awk -F':' {'print $3'} | tr -d '/')
+
+export https_proxy_host=$(echo $https_proxy | awk -F':' {'print $2'} | tr -d '/')
+export https_proxy_port=$(echo $https_proxy | awk -F':' {'print $3'} | tr -d '/')
+
+export _JAVA_OPTIONS="-Dhttp.proxyHost=$http_proxy_host -Dhttp.proxyPort=$http_proxy_port -Dhttps.proxyHost=$https_proxy_host -Dhttps.proxyPort=$https_proxy_port"
+
+REMOTE_URL=https://github.com/secure-device-onboard/pri.git
+REMOTE_BRANCH=master
+
+if [ "$use_remote" = "1" ];
```

Comment:
Update indent as per review comments in other PR.

Reply:
Changed indentation to 2-space format.